### ISSUE TITLE
Eternal Wallet Creation/No Synch/

### ISSUE THREAD

**Issue by 0-0-0-** [ethereum/mist#3293]
- [ x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

Version: `0.9.2`
OS & Version OSX 10.13.1
Node version: geth 1.7.2 
Number of blocks synchronized: Claims 4,561335@ 0427am GMT + Counting up in agreement with
Etherscan
[node copy.log.zip](https://github.com/ethereum/mist/files/1477338/node.copy.log.zip)
[Archive.zip](https://github.com/ethereum/mist/files/1477340/Archive.zip)

```


<img width="1163" alt="screen shot 2017-11-16 at 04 33 33" src="https://user-images.githubusercontent.com/7877399/32874060-b6a100d0-ca87-11e7-9aa7-9fe55a961851.png">

Tried to create a new wallet,but its being eternally created and even though the node claims to be updated the balance is not updating. System clock is correct and block count is inline with Etherscan.
Not a developer so my only option is delete and start over Unless there is some help here

 -->

**Comment by 0-0-0-** [comment]
Deleted the client and reinstalled. Same issue persists. Wallet is still being created.No peers being shown now. Balance updated but still no normal operation in terms of peers.

**Comment by evertonfraga** [comment]
This is a known issue and the fix has already been merged. It will be released in the next version, 0.9.3.

Refer to this issue for discussion: #3269.

Thanks.
